chingu-coders/Voyage2-Turtles-02
# fixed clock memory leak

## Purpose
* Standard clock and pomodoro clock both had memory leaks realted to updating the text content of the display

## Approach/Fixes
* Identified `toLocaleString` and `textContent` as leaky methods. Replaced with my own function, and `innerText` respectively.

## Testing
* Check out the pull request
* open the app with chrome devtools. In the performance tab run `record` for about 30 seconds, both with and without a pomodoro cycle running
* Confirm that the `nodes` graph doesnt skyrocket. jajaja
